* Corrected documentation regarding database.

nolantellis opened the pull request:
* Corrected code to include new liquibase user.
* corrected documentation

Commit:
* Corrected documentation regarding database.
* Corrected code to include new liquibase user.

Commit:
Correcting test property file

brucenguyen1 commented on site.properties:
```diff
@@ -89,29 +89,35 @@ ldap.lastNameAttribute  = sn
-jdbc.url      = jdbc:mysql://localhost:3306/apromore?createDatabaseIfNotExist=true&autoReconnect=true&allowMultiQueries=true&rewriteBatchedStatements=true&characterEncoding=utf-8
-jdbc.username = apromore
-jdbc.password = MAcri
+#jdbc.driver   = com.mysql.jdbc.Driver
+#jdbc.url      = jdbc:mysql://localhost:3306/apromore?createDatabaseIfNotExist=true&autoReconnect=true&allowMultiQueries=true&rewriteBatchedStatements=true&characterEncoding=utf-8
+#jdbc.username = root
+#jdbc.password = MAcri
+#
+#jpa.database         = MYSQL
+#jpa.databasePlatform = org.eclipse.persistence.platform.database.MySQLPlatform
+#jpa.showSql          = false
+#jpa.generateDDL      = false
```
Why don't we use the same 'apromore' user like H2 instead of 'root'?

brucenguyen1 replied:
Can it is named "liquibase_user" instead of "db_user" (it is full privileges)? The name will be shown in the database management tool for information as well.

nolantellis replied:
I am not sure how it became root. Changeing that back to apromore. It was a mistake.

db_user is just a name. And looking at the Docs of mysql about the privileges, All privileges is the right 1. (i.e ALL privileges does not include create user.)

nolantellis replied:
Corrected to liquibase_usr

brucenguyen1 commented on site.properties:
```diff
@@ -89,29 +89,35 @@ ldap.lastNameAttribute  = sn
-jpa.databasePlatform = org.eclipse.persistence.platform.database.MySQLPlatform
-jpa.showSql          = false
-jpa.generateDDL      = false
+## Liquibase User Mysql
+#liquibase.jdbc.username = db_user
+#liquibase.jdbc.password = 7fHJV41fpJ
 
```
'7fHJV41fpJ': is it an encrypted value of the password or is it the password? If it is the password, then it should be encrypted as it can be seen easily in this file (it is full privileges).

nolantellis replied:
Its just a random passowrd. encryption is a different logic which we can consider later. Its not in this scope

brucenguyen1 replied:
Ok if it's a low security risk for showing password of an "All privileges" user in a text file. Any justification?

nolantellis replied:
The use is excactly the same as apromore user at this time.
All previleges is only for apromore db, and as per the mysql5.7 doc All privilege is DML and DDL and not USER management.

I Agree this is not the right way, but thats another scope to consider not only for liquibase but also for apromore user.

brucenguyen1 replied:
https://dev.mysql.com/doc/refman/5.7/en/grant.html (Table 13.8, the "Level" information). There's a risk of losing data due to password is revealed in plain text file for 'apromore' and 'liquibase_user', at the same risk level for both users. This risk should be addressed in the future if possible.

Commit:
corrected root to apromore

Commit:
Minor changes in liquibase user

Commit:
Corrected Readme docs

raboczi committed:
Update database-jpa.xml

Two elements with same @id ("dataSource") in the same XML file is confusing, so removed it from the embedded bean.

Commit:
Merge remote-tracking branch 'origin/task/update-liquibase-doc/AP-2484' into task/update-liquibase-doc/AP-2484

raboczi reviewed (approved):
I'm satisfied that it starts properly with zero configuration under H2, and that it starts properly under MySQL given the setup described in README.md.
